spec/bitbot/responder_spec.rb

```diff
@@ -37,6 +37,26 @@
     it "returns false if there's no route for the message" do
       expect(described_class.responds_to?(message)).to be_falsy
     end
+
+    context "whitelisting" do
+      it "returns false if the route isn't in the whitelist" do
+        described_class.route(:test, /lana/) {}
+        described_class.route(:secure, /LANA/) { 'Secure route' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.